tools/topology/topology2/platform/intel/google-rtc-aec-generic.conf

```diff
@@ -0,0 +1,156 @@
+Object.Pipeline.gain-module-copier [
```

Comment:
context on what `generic` means?

Reply:
it means it will fit for all google rtc aec usage

Reply:
That still isn't clear how it differs from the capture

Reply:
A pcm stream is composed of at least two pipelines,  one for input processing and the other is for output processing. google-rtc-aec-capture.conf is for output processing.  generic.conf includes pcm definition, how to connect to input processing pipeline and route info.

Reply:
> google-rtc-aec-capture.conf is for output processing

That is very confusing

Why no call them capture (actually capture) and reference?

Reply:
good idea. I will change it.